yellowbrick/cluster/elbow.py

```diff
@@ -247,6 +252,9 @@ def fit(self, X, y=None, **kwargs):
 
         self.k_scores_ = []
         self.k_timers_ = []
+        self.kneedle=None
```

Comment:
Does the `KneeLocator` store any state that we may need to keep around? If so, we should store it in a variable `self._knee_locator` (just to make things obvious to other YB contributors). If not, there is probably no reason to keep this object on the visualizer, simply make it a variable local to `fit()` which will be discarded when we're done fitting.

Reply:
@bbengfort For the moment I don't think there's any need to include `knee locator` object in the visualizer. We can make it local to `fit()`. Also we have to name it ` elbow_locator `  right?

Reply:
I agree, also after more consideration; the `KneeLocator` object holds all the data, which is something we don't want to do generally, so I think it's good that we're not storing it on the Visualizer. I'm fine with either `knee_locator` or `elbow_locator` -- your choice!